<launch>


<param name="is_sim" value="True" />


<group ns="g2">

<param name="port" value="8002" />
<param name="id" type="str" value="2" />

<node name="perceptor" pkg="drcs" type="perceptor_launcher.py">
  </node>
<node name="analyser" pkg="drcs" type="analyser_launcher.py">
  </node>
<node name="declamator" pkg="drcs" type="declamator_launcher.py">
  </node>
<node name="translocator" pkg="drcs" type="translocator_launcher.py">
  </node>
<node name="mover" pkg="drcs" type="mover_launcher.py">
  </node>
<node name="navigator" pkg="drcs" type="navigator_launcher.py">
  </node>
<node name="positionator" pkg="drcs" type="positionator.py">
  </node>

</group>


</launch>